cmd/out/main.go

```diff
@@ -49,6 +53,27 @@ func main() {
 		blobPath := filepath.Dir(outRequest.Source.Regexp)
 		blobBaseName := filepath.Base(outRequest.Params.File)
 		blobName = filepath.Join(blobPath, blobBaseName)
+
+		matcher, err := regexp.Compile(outRequest.Source.Regexp)
+		if err != nil {
+			log.Fatal("failed to compile source configuration regex: ", err)
+		}
+
+		matches := matcher.FindStringSubmatch(blobName)
+		if len(matches) >= 2 {
```

Comment:
I wasn't sure if `len(matches) < 2` if the regex doesn't match anything in the blob name should be a fatal error, or if this implementation was sufficient.

Reply:
I think not failing here is okay. If it causes unforeseen issues to not explicitly error we can add it in later, but I don't expect this to happen. It also keeps old behavior (i.e didn't use to error if the regex didn't find a match), so in that regard it may also be valuable.